apiary.apib

```diff
@@ -304,32 +304,113 @@ A list of objects containing the following parameters:
 |  Name | Description | Example |
 | ---------------| ------------| --------|
 | session_id | unique ID of the session | fOBg4dXAPTHSoil6LywgfYA |
+| id | Always the same as session_id, left for backwards compatibility | fOBg4dXAPTHSoil6LywgfYA |
```

Comment:
i think adding field is ok too, the question was mostly about a description. how is that left for backwards comparability if it is a new field? 😕